helix-term/src/ui/editor.rs

```diff
@@ -717,9 +717,9 @@ impl EditorView {
 
         for line in view.offset.row..(last_line + 1) {
             let area = Rect::new(
-                view.area.x,
-                view.area.y + (line - view.offset.row) as u16,
-                view.area.width,
+                view.inner_area().x,
+                view.inner_area().y + (line - view.offset.row) as u16,
+                view.inner_area().width,
```

Comment:
`let inner_area = view.inner_area()` before this line. Otherwise you're calculating it three times.